vagrant/lib/forklift/box_distributor.rb

```diff
@@ -286,12 +289,16 @@ def configure_rackspace(machine, box)
 
     def configure_openstack_provider(machine, box)
       machine.vm.provider :openstack do |p, override|
-        override.vm.box        = nil
-        override.ssh.username  = 'root'
+        override.vm.box = nil
+        if box.fetch('sync_type', 'disabled')
+          override.vm.synced_folder '.', '/vagrant', disabled: true
+        end
         override.ssh.pty       = true if box.fetch('pty', nil)
+        override.ssh.username  = box.fetch('username', 'root')
         p.server_name          = machine.vm.hostname
-        p.flavor               = /4GB/
+        p.flavor               = box.fetch('openstack_flavor', nil)
```

Comment:
@ekohl will we need to adjust the config we use on Rackspace due to this?

Reply:
Yes, this will break our CI. I'd prefer to keep the `/4GB/` default instead of `nil`. Since the flavor is required I don't think this should be an issue.